### ISSUE TITLE
Visual style

### ISSUE THREAD

**Issue by amorriscode** [hyshka/moonshot#6]
I wanted to have a discussion about the visual/UI style.

## Approach

1. Lighthearted (think pixel art or hand-drawn looking)
2. Serious (think UI from sci-fi films, etc.)

## Serious UI, Lighthearted Game

We could take the approach of everything being lighthearted but since the game itself is lighthearted/silly, what if we juxtaposed that with serious UI? People would expect the game to be super serious when it boots up but then they get a surprise when they play.

We could implement React and use this: https://arwes.dev/. However, I'm not sure if that would provide the best flexibility. I'm happy to tackle the UI. If I find that Arwes isn't flexible enough (glitching, etc.) then I will just use Tailwind and build out stuff inspired by it.

What do you think?

**Comment by hyshka** [comment]
I love the arwes UI, I think that style would be perfect, I love the sounds and animations as well.

When it comes to visual style I think there are a few components:

- The UI (which I think is pretty much nailed by arwes), this is our menu systems, dialogue, etc.
- Game art, this is character sprites, actual objects moving around in the game, space ship or something, backgrounds, etc.
- User feedback. I'm tempted to combine this into UI but what I'm thinking of here is more along the line of what happens in response to user input during gameplay. For example, if you're about to crash is there a flashing button on your HUD or does the character black out or something? Also, when considering the gameplay loop there may be other things either in a HUD or maybe ship controls we want to display.

UI is great here but can you give some examples of pixel art or the style you have in mind there?

P.S. I love the idea of having lighthearted sprites or game art but a serious UI or HUD

**Comment by amorriscode** [comment]
@hyshka here are some resources.

- My preference for planets: https://helianthus-games.itch.io/pixel-art-planets
- Characters (might be useful somehow): https://penusbmic.itch.io/sci-fi-character-pack-6
- Icons (for splash screens, etc.): https://acidsilk.itch.io/space-rpg-icons
- Possible music: https://gooseninja.itch.io/space-music-pack
- Explosions (and maybe spaceship): https://ansimuz.itch.io/spaceship-shooter-environment

I'll be honest I haven't found any spaceships I'm super keen on but this looks like a great start.

I could likely commission a pixel ship if we had some sort of idea what we'd want it to look like!

**Comment by hyshka** [comment]
Sweet! That gives me a good feel for the pixel art. I really like those planets!

How do you see it taking on a "lighthearted" aesthetic? To me, it almost seems like that relies a bit more on dialogue or situational humour to express. What are your thoughts?

**Comment by amorriscode** [comment]
Yeah I agree. We could have the Delivery Person have an inner dialogue that provides humour, etc. Plus the situational humour (packages he is delivering, etc.).

**Comment by hyshka** [comment]
A co-pilot might also be cool as that could provide a lot more dialogue opportunities, the co-pilot could criticize the player or provide suggestions.
